documentation/manifest.md

```diff
@@ -82,6 +82,15 @@
         - [ ] Progress bar appears above Synchronization title
         - [ ] Team Explorer view changes to Home page
         - [ ] Notification appears: "Repository published successfully"
+        - [ ] Publish a private repo and verify on .com that it's private 
+- [ ] **Project section (Home button)*** 
+    - [ ] Click on "Home" icon 
+        - [ ] Verify Project has the following sections/buttons: Changes, Pull Requests, Branches, Sync, Pulse, Graphs, Wikis, Issues, Wikis, and Settings
```

Comment:
👍 thanks @shana. I'll clarify in the manifest that Wikis and Issues _should_ display when you're signed into github and the repo has it enabled.